Kickstarter-iOS/AppDelegate.swift

```diff
@@ -191,7 +191,7 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
       .observeForUI()
       .observeValues { [weak self] in self?.findRedirectUrl($0) }
 
-    //swiftlint:disable discarded_notification_center_observer
+    // swiftlint:disable discarded_notification_center_observer
```

Comment:
For consistency with the rest of the codebase `// swiftlint` is preferred to `//swiftlint`